main.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'app'
+
+def main
+  app = App.new
+
+  puts 'Welcome to School Library App!'
+
+  options = {
+    'List all books' => :list_books,
+    'List all people' => :list_people,
+    'Create a person' => :create_person,
+    'Create a book' => :create_book,
+    'Create a rental' => :create_rental,
+    'List all rentals for a given person id' => :list_rentals_by_person_id,
+    'Exit' => :exit
+  }
```

Comment:
- [ ] Good job so far But I noticed that your menu starts with 0 instead of 1 as you can see in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided to you in the project requirement. Please fix it :+1: